templates/zerver/help/include/sidebar_index.md

```diff
@@ -81,7 +81,7 @@
 * [Emoji reactions](/help/emoji-reactions)
 * [Star a message](/help/star-a-message)
 * [View and browse images](/help/view-and-browse-images)
-* [Link to a message or conversation](/help/link-to-a-message-or-conversation)
+* [Link to a message or topic](/help/link-to-a-message-or-conversation)
```

Comment:
Hm, could you explain your thinking with this change?

Reply:
The sidebar lists the help file under "Streams and Topics." The menus for streams and topics use the "topic" label. In the zulip/help folder, "conversation" is used only 59 times; "topic" 555 times. A search of Zulip code also indicates that "topic" is the term preferred by developers. A discussion in the documentation stream indicates that "conversation" is a somewhat nebulous term, while "topic" seems a bit better understood.

I'm agnostic about which term is used, but a stickler for consistency. I think using "conversation" is confusing, particularly when referencing UI elements and functionality that is exclusively labelled as "topic" in the UI.

https://chat.zulip.org/#narrow/stream/19-documentation/topic/Topic.20vs.20Conversation

Reply:
In particular, I find https://chat.zulip.org/help/link-to-a-message-or-conversation to be particularly grating: while the help is labelled "Link to a message or conversation," there are no subheads that use the term "conversation" and, indeed, the word is only used in the introductory sentences while the rest of the page uses the term "topic" exclusively. 

As a naïve user, I would find it very confusing to use this help.